src/Spout/Common/Escaper/ODS.php

```diff
@@ -22,7 +22,15 @@ class ODS implements EscaperInterface
      */
     public function escape($string)
     {
-        return htmlspecialchars($string, ENT_QUOTES);
+        if (defined('ENT_DISALLOWED')) {
+            return htmlspecialchars($string, ENT_QUOTES | ENT_DISALLOWED);
+        } else {
+            // We are on hhvm or any other engine that does not support ENT_DISALLOWED
+            // https://github.com/box/spout/issues/329
+            $escapedString =  htmlspecialchars($string, ENT_QUOTES);
+            $replacedString = preg_replace('/[\x00-\x08\x0B-\x0C\x0E-\x1F]/', '�', $escapedString);
```

Comment:
can you define a extra variable for the regex pattern. Similar to what's done for the XLSX escaper:

```php
// control characters values are from 0 to 1F (hex values) in the ASCII table
// some characters should not be escaped though: "\t", "\r" and "\n".
return '[\x00-\x08' .
        // skipping "\t" (0x9) and "\n" (0xA)
        '\x0B-\x0C' .
        // skipping "\r" (0xD)
        '\x0E-\x1F]';
```